src/docs/blueprint_documentation.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Packaging Theia as a Desktop Product
+---
+
+# Packaging Theia as a Desktop Product
+
+Eclipse Theia Blueprint product is a template to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform.
```

Comment:
Throughout the documentation we refer to the product differently, we should be consistent in the naming.
For instance I see:
- `Eclipse Theia Blueprint`
- `Theia Blueprint`
- `Theia blueprint`

The first mention can be `Eclipse Theia Blueprint` (as per the foundation rules), but subsequent references should be `Theia Blueprint` and not `Theia blueprint`.

Reply:
First mention is `Eclipse Theia Blueprint` now, after that we use `Theia Blueprint`